repose-aggregator/services/datastore/impl/src/main/java/org/openrepose/nodeservice/distributed/servlet/DistributedDatastoreServlet.java

```diff
@@ -97,7 +100,12 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
             if ("PATCH".equals(request.getMethod())) {
                 doPatch(request, response);
             } else {
+                String traceGUID = request.getHeader(CommonHttpHeader.TRACE_GUID.toString());
```

Comment:
Probably just because derp. This is easy to solve. I fix. 

On May 21, 2015 7:52:26 AM CDT, Adrian J George III <notifications@github.com> wrote:
>> @@ -97,7 +100,12 @@ protected void service(HttpServletRequest
>request, HttpServletResponse response)
>>              if ("PATCH".equals(request.getMethod())) {
>>                  doPatch(request, response);
>>              } else {
>> +                String traceGUID =
>request.getHeader(CommonHttpHeader.TRACE_GUID.toString());
>
>we use patch a lot, why did you not include it?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/rackerlabs/repose/pull/1306/files#r30798192

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.